token/services/db/sql/init.go

```diff
@@ -20,28 +20,30 @@ import (
 
 var logger = flogging.MustGetLogger("token-sdk.sql")
 
-func initSchema(db *sql.DB, schemas ...string) error {
+func initSchema(db *sql.DB, schemas ...string) (err error) {
```

Comment:
Since you always return the error, you don't have to declare the result var in the signature.